compiler/luci/import/src/CircleReader.cpp

```diff
@@ -257,33 +276,46 @@ std::unique_ptr<SparsityParam> luci_sparsityparam(const circle::SparsityParamete
   return sparsityparam;
 }
 
-void copy_tensor_attributes(const circle::TensorT &tensor, CircleNode *node)
+std::unique_ptr<SparsityParam> luci_sparsityparam(const circle::SparsityParameters *sparparam)
```

Comment:
why is this added?

Reply:
Sparsity param needed above:

https://github.com/Samsung/ONE/blob/eda824dab4f2e547760de161e9f935177796f33c/compiler/luci/import/src/CircleReader.cpp#L318-L321

`copy_tensor_attributes` removed because this function unused.